src/action/memory_size.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file or at https://opensource.org/licenses/MIT.
+
+//! A handler and associated types for the `GetMemorySize` action.
+//!
+//! The `GetMemorySize` action returns the RAM size in bytes.
+
+use sysinfo::SystemExt;
+use crate::session::{self, Session};
+
+pub struct Response {
+    memory_size: u64
+}
+
+pub fn handle<S: Session>(session: &mut S, _: ()) -> session::Result<()> {
+    let mut system = sysinfo::System::new();
+    system.refresh_all();
```

Comment:
Why `refresh_all`? I don't think you care about *all* system information.

Reply:
Fixed